contracts/utils/Helpers.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.4 <0.9.0;
+
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+/**
+ * @dev Library of helpers for generating SQL statements from common parameters.
+ */
+library Helpers {
+    /**
+     * @notice Generates a properly formatted table name from a prefix and table id.
+     * @param prefix the user generated table prefix as a string.
+     * @param tableId the Tableland generated tableId as a uint256.
+     * @return name newly allocated string containing the table name.
+     *
+     * @dev requirements: block.chainid must refer to a supported chain.
```

Comment:
maybe this is the new recommended style (not sure), but the rest of the code base uses a different convention: https://github.com/tablelandnetwork/evm-tableland/blob/main/contracts/ITablelandTables.sol#L102. i think we should be consistent throughout.

Reply:
Fwiw, I started down the path of inline docs standards; haven't finished / requested everyone's input on it yet. But, through that process and looking at misc. contracts, I found that `@dev`, `@param`, and `@return` were commonly used as they follow [Natspec](https://docs.soliditylang.org/en/v0.8.15/natspec-format.html) and help for Solidity compiler purposes (tag interpretation). The next most common Natspec recommendations were `@title` and `@notice` and, following that, the rarely used but helpful `@inheritdoc`.

All "nice to haves" since not many contracts seemed to use these, even though Solidity / Natspec recommends it -- I was thinking `@dev`, `@param`, and `@return` would maybe be the minimum standard.

Namely, it's super inconsistent across codebases, e.g., [ERC721A](https://github.com/chiru-labs/ERC721A/blob/main/contracts/ERC721A.sol) and many of OZ's contracts don't follow the Natspec very well and more closely align to what `evm-tableland` currently does. Or, [contracts will use both styles](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC721/extensions/ERC721Enumerable.sol) without a clear reason why they differ. And then something like Uniswap ([here](https://github.com/Uniswap/v3-core/blob/main/contracts/libraries/FullMath.sol)  and [here](https://github.com/Uniswap/v3-core/blob/main/contracts/interfaces/IUniswapV3Factory.sol)) goes all in and uses all of the tags mentioned above.

https://www.notion.so/textile/Docs-Format-Standard-WIP-0f3e494265ce4f8b91588da993892392#1a36a3d5de7743d8a0737b90403a2277

Reply:
ah yeah nice, i had gone by https://docs.soliditylang.org/en/v0.8.15/natspec-format.html too to figure out what to put in here.

Reply:
yeah, there's no strong convention / consensus on this, probably because there isn't something like godoc / jsdoc / javadoc that spits out a nice docs page from solidity doc strings (yet). our current style is copied from OZ and ERC721A convention. Let's stick with that. i find it more readable actually, and is already in place.

Reply:
sgtm

Reply:
ah yeah... it's not so easy. i had looked for the "right" way because i saw two ways in the code base. e.g. 

`baseURITemplate - a string containing "{id}"` in URITemplate
versus
`@param needle The text to search for in `self`.` in SStrings. 

i think what you want is the former. so something like,

```
    /**
     * @dev Returns a token URI based on the set template string.
     *
     * tokenIdStr - the tokenId as a string
     */
```

where `@dev` is the top level flag. drop the `@param` flag? i was looking for an example of the '@returns' and don't see an example in this code base. drop it and just include in dev text maybe? 

i did notice that OZ isn't consistent, so was looking for how they do returns too and it's fuzzy https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC721/ERC721.sol#L386

maybe worth highlighting just one super flushed out doc string in the code base so all others can conform to it. maybe it's this one https://github.com/tablelandnetwork/evm-tableland/blob/main/contracts/policies/ERC721EnumerablePolicies.sol#L16 ? return in dev string, no params (doesn't have that requirements usage, but minor)

Reply:
the ref I linked above is a good (https://github.com/tablelandnetwork/evm-tableland/blob/main/contracts/ITablelandTables.sol#L102). @dev as top level flag, no @params or @returns.

`SStrings` is copied from an external (licensed) project, so I didn't change anything.

we can revisit this in a TPD / other PR is we want to make changes to the current code base convention.... and then document it, etc.

Reply:
perfect, updating.